ragna/deploy/_cli/core.py

```diff
@@ -131,7 +131,10 @@ def check_api_available() -> bool:
     try:
         if process is not None:
 
-            @timeout_after()
+            @timeout_after(
+                60,
+                "Failed to start the API in 60 seconds. Please start it manually with ragna API.",
```

Comment:
The lower casing here was intentional as it refers to the command the user should run.

```suggestion
                "Failed to start the API in 60 seconds. Please start it manually with ragna api.",
```

That being said, I dislike that it looks exactly like the other text and thus is hard to interpret. One idea could be to make it bold. What do you think? I can guide you through the process if you are up to it.

Reply:
I will change the capitalization.

Bold is one good idea. As another example, git puts commands in quotes like 

> `(use "git push" to publish your local commits)`

to indicate commands. I think bolded with quotes would be a good idea because the bold draws attention, and quotes explicitly tell the user what command to run. 

Your guidance for the bold text would be appreciated.

Reply:
Both quoting the command or making it bold add emphasis and I prefer not to have two ways of doing the same. To make it bold, we need print it with `rich`. To be able to do that, we first need to except the `TimeoutError`

```python
try:
    wait_for_api()
except TimeoutError:
    rich.print(
        "Failed to start the API in 60 seconds. "
        "Please start it manually with [bold]ragna api[/bold]."
    )
    raise typer.Exit(1)
```

With this we can also remove the custom message from the decorator since we ignore it anyway.

Reply:
Let me know what you think of the new changes